space/space.go

```diff
@@ -54,6 +55,7 @@ type Repository interface {
 	Load(ctx context.Context, ID satoriuuid.UUID) (*Space, error)
 	Delete(ctx context.Context, ID satoriuuid.UUID) error
 	List(ctx context.Context, start *int, length *int) ([]*Space, uint64, error)
```

Comment:
Well, the `List` method is in the Repository interface, not in a Controller. The Repository could be used in other services/controllers as well, so we should look at the multiple controllers that would use this method, and then alter the method behaviour based on the needs of the API's clients.